request_handler.py

```diff
@@ -0,0 +1,75 @@
+import importlib
+import methods
+import copy
+
+def execute(request):
+    """
+    Executing requests using the methods defined in the methods module.
+    
+    Request dict format:
+    {"func" : <function name that is in the methods module>,
+     "requests" : <list of dicts, functions and arguments to run in order. Not used if "func" is not equal to "tether">, 
+     "args" : <ordered arguments for func, including data input. For tether, these are the initial input arguments>,
+     "kwargs" : <key worded arguments for func>
+    }
+    
+    input: request (dict)
+    output: <tether function> OR method.func(*args, **kwargs)
+    """
+    
+    args = []
+    kwargs = {}
+    
+    if "args" in request:
+        if isinstance(request["args"], type(None)):
+            pass
+        else:    
+            args = request["args"]
+    
+    if "kwargs" in request:
+        if isinstance(request["kwargs"], type(None)):
+            pass
+        else:    
+            kwargs = request["kwargs"]
+        
+    func_name = request["func"]
+    if func_name == "tether":
+        return tether(request["requests"], args, kwargs)
+    
+    #print("%s.%s"%("methods",func_name))
+    #print("args:", args, "kwargs:", kwargs)
```

Comment:
It would be better to import your methods up front, rather than in the middle of a request cycle, since an import might be expensive. I would have a single module 'methods' or 'functions' (probably 'functions' to agree with the 'func' nomenclature) which imports everything up top; then check for the presence of an appropriately-named function here before executing.

Reply:
You mean when the server begins right (like the functions module is loaded along with flask and other things)? That makes sense, I was thinking for heavier imports that may never be used in a call or conflicting imports it may be an issue but that is probably in a minority of cases